projects/vdk-plugins/vdk-ipython/src/vdk_ipython.py

```diff
@@ -38,8 +88,5 @@ def magic_load_job(line: str):
 def load_job(
     path: str = None, name: str = None, arguments: str = None, template: str = None
 ):
-    path = pathlib.Path(path) if path else pathlib.Path(os.getcwd())
-    with StandaloneDataJobFactory.create(
-        data_job_directory=path, name=name, job_args=arguments, template_name=template
-    ) as job_input:
-        get_ipython().push(variables={"job_input": job_input})
+    job = JobControl(path, name, arguments, template)
+    get_ipython().push(variables={"data_job": job})
```

Comment:
I was thinking. Wouldn't be better to be "VDK" ? 


Like 

```
data_job.get_initialized_job_input()
```

or 
```
VDK.get_initialized_job_input()
```

We can add more stuff related to VDK in the future. VDK is the more familiar and user facing name . That's why. 

Similarly to how click project exposes a lot of interfaces using `click.xxx` interface (click is the name of the project and main entry point)

Reply:
I like this naming more, as well